src/app/components/Layout.tsx

```diff
@@ -25,7 +25,13 @@ export const wrapperPadding = css`
 // tslint:disable-next-line:variable-name
 export const LayoutBody = styled.div`
```

Comment:
these changes are breaking the dev homepage, in this place, `/app/components`, we shouldn't be making assumptions that the content will follow the content layout, the grid described here is content specific and should be defined  as the content layout